pkg/server/grant/grant.go

```diff
@@ -195,7 +195,7 @@ func (l *Grant) handleGrant(user user.Info, w http.ResponseWriter, req *http.Req
 		return
 	}
 
-	req.ParseForm()
```

Comment:
wouldn't it make sense to handle the error here?

Reply:
sorry, I just blindly implemented the obvious, not any handling logic, let me do so.

Reply:
@stlaz, in `handleGrant` we don't return error http status codes at all, should I add them there? :sweat_smile: